csm/conf/csm_cleanup.py

```diff
@@ -79,38 +79,41 @@ def remove_old_indexes(es, arg_d, arg_n, arg_e):
         else:
             Log.debug("Nothing to remove")
 
-def process_stats(args):
-    # Pass arguments to worker function
-    es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    remove_old_indexes(es, args.d, args.n, args.e)
 
-def process_auditlogs(args):
+def process_es_cleanup(args):
     # Pass arguments to worker function
     # remove data older than given number of days
     es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    es.remove_old_data_from_indexes(args.d, args.n, args.i, args.f)
+    es.remove_old_data_from_indexes(args.d, args.n, ["csmauditlog"], "timestamp")
```

Comment:
extract it as a function... add logs in debug

Reply:
Comment addresed. Please review